ui/src/main.rs

```diff
@@ -992,7 +1006,7 @@ fn app(cx: Scope) -> Element {
     cx.render(main_element)
 }
 
-fn get_pre_release_message(cx: Scope) -> Element {
+fn get_pre_release_message(cx: &ScopeState) -> Element {
```

Comment:
changing `get_pre_release_message` to take a `ScopeState` will conflict with a branch I'm working on. Long story short is in order to use `use_shared_state` in this function, I had to create it like this: `get_pre_release_message{}` instead of like this: `get_pre_release_message(cx)`.